RealEstate/urls.py

```diff
@@ -29,6 +29,7 @@
         'django.contrib.auth.views.logout_then_login', name='auth_logout'),
     url(r'^change-password/$', 'django.contrib.auth.views.password_change',
         {'post_change_redirect': 'home'}, name='password_change'),
+    url(r'^homebuyerreport/$',PendingViews.HomebuyerReportView.as_view(),name='report'),
```

Comment:
There will be a homebuyer report for each couple. Take a look at how this is done for house evals. Needs more than just static endpoint. The name 'report' will be used in your link above. Again, follow the eval model.